### PR TITLE
Create 2024-02-23-BirForex.md

### PR DESCRIPTION
**Pull request by svg-arch**

### CODE REVIEW

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+date: 2024-02-23
+target-entities: BitForex
+entity-types:
+  - Exchange
+  - Custodian
+attack-types:
+  - Exit Scam
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:8]: Change to `Scam` to match other attack types

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - Wallet Hack
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:9]: Remove the line

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+title: "BitForex's Suspicious Outflows: Hack or Exit Scam?"
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:10]: Rewrite the title in the affirmative

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+loss: 56500000
+---
+
+## Summary
+
+Since February 23, 2024, BitForex, a crypto exchange operational since 2017, ceased processing withdrawals amidst unexplained outflows of about $56.5M worth of crypto from its hot wallets. The absence of communication from BitForex, coupled with the recent departure of its CEO Jason Luo, has raised concerns over a potential inside job or exit scam. 
+
+## Attackers
+
+The identity of the attacker is unknown. The following addresses are associated with this attack:
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:20]: You already mentioned CEO of BitForex, you can mention him here. Preferably with sources with official accusations.

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    
+- Tron Wallet:
+  - [TQcnqaU4NDTR86eA4FZneeKfJMiQi7i76o](https://tronscan.org/#/address/TQcnqaU4NDTR86eA4FZneeKfJMiQi7i76o)
+   
+- Bitcoin Wallet:
+  - [3DbbF7yxCR7ni94ANrRkfV12rJoxrmo1o2](https://www.blockchain.com/explorer/addresses/btc/3DbbF7yxCR7ni94ANrRkfV12rJoxrmo1o2)
+
+## Losses
+
+The total loss from the BitForex hack amounted to approximately [$56.5 million](https://twitter.com/zachxbt/status/1762028433574650347), distributed across the following networks:
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:33]: There is no evidence that it was a hack. It's more likely to be an exitscam. [This source](https://bits.media/gonkongskiy-regulyator-obvinil-kriptobirzhu-bitforex-v-moshennichestve/) might be useful.

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  - 44,000 TRX (6072 USD)
+
+- Bitcoin
+  - 5.7 BTC (290,000 USD)
+
+## Timeline
+
+- **January 31, 2024, 02:07 PM UTC:** Exchange CEO Jason Luo [steps down](https://support.bitforex.com/hc/en-us/articles/28260960127385-Jason-Stepped-Down-as-CEO-of-BitForex).
+- **February 21, 2024, 08:16:23 AM UTC:** The [first malicious](https://etherscan.io/tx/0xee8a0e425670b53e5066451362324fb6ef36fb5507a525567fc8c9b68a03709f) transaction occurred.
+- **February 23, 2024, 07:12:59 PM UTC:** The withdrawal of funds from the exchange [began](https://etherscan.io/tx/0xb1ef6176112c8b019192b5e627732afee8930810801961ded081720e74544753) and the exchange stopped operating without warning.
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:52]: Unnecessary line. You can add a source that posted about website shutdown instead.

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+- **February 26, 2024:** 0xScope [published](https://www.0xscope.com/blog-posts/hack-or-exit-scam-what-happened-at-bitforex) an analysis of the incident. 
+
+## Security Failure Causes
+
+- **Inside Job:** The nature of the outflows, characterized by premeditated transactions and a significant delay in response from BitForex, suggests this may not be a straightforward hack. Initial transactions, such as a test transaction of 0.01 $ETH followed by a 58-hour inaction, indicate a controlled extraction of funds rather than a hurried hack.
+- **Wallet System Vulnerability:** Flaws in Bitrue’s hot wallet system have led to unauthorized access and transfers of funds.
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:58]: Bitrue?

**Review by JediFaust — CHANGES_REQUESTED** [review]
Overall, we have no evidence on BitForex performed exitscam. So, I suggest to wait till it's officially confirmed.

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 8]
```diff
@@ -48,11 +49,12 @@ The total loss from the BitForex hack amounted to approximately [$56.5 million](
...
-- **February 23, 2024, 07:12:59 PM UTC:** The withdrawal of funds from the exchange [began](https://etherscan.io/tx/0xb1ef6176112c8b019192b5e627732afee8930810801961ded081720e74544753) and the exchange stopped operating without warning.
-- **February 26, 2024:** 0xScope [published](https://www.0xscope.com/blog-posts/hack-or-exit-scam-what-happened-at-bitforex) an analysis of the incident. 
+- **February 21, 2024, 08:16 AM UTC:** The [first malicious](https://etherscan.io/tx/0xee8a0e425670b53e5066451362324fb6ef36fb5507a525567fc8c9b68a03709f) transaction occurred.
+- **February 23, 2024:** [The exchange website has closed.](https://decrypt.co/219012/exit-scam-bitforex-shutters-after-57-million-withdrawn)
+- **February 26, 2024:** 0xScope [published](https://www.0xscope.com/blog-posts/hack-or-exit-scam-what-happened-at-bitforex) an analysis of the incident.
+- **March 4, 2024:** Hong Kong’s regulator for securities and futures markets has [warned](https://www.coindesk.com/policy/2024/03/04/hong-kongs-markets-regulator-issues-warning-against-crypto-exchange-bitforex) the public about BitForex for suspected fraud.
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:55]: Add original source from SFC

File: content/attacks/posts/2024-02-23-BitForex.md [hunk 9]
```diff
@@ -48,11 +49,12 @@ The total loss from the BitForex hack amounted to approximately [$56.5 million](
...
 
 ## Security Failure Causes
 
 - **Inside Job:** The nature of the outflows, characterized by premeditated transactions and a significant delay in response from BitForex, suggests this may not be a straightforward hack. Initial transactions, such as a test transaction of 0.01 $ETH followed by a 58-hour inaction, indicate a controlled extraction of funds rather than a hurried hack.
-- **Wallet System Vulnerability:** Flaws in Bitrue’s hot wallet system have led to unauthorized access and transfers of funds.
+- **Wallet System Vulnerability:** Flaws in the BitForex hot wallet system have led to unauthorized access and transfers of funds.
```
Comment (JediFaust) [inline content/attacks/posts/2024-02-23-BitForex.md:60]: Remove the line, since it's not a hack

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by evgenydmitriev — APPROVED** [review]

**Comment by svg-arch** [comment]
@evgenydmitriev x2 for Custodian?

**Comment by evgenydmitriev** [comment]
@jhirschkorn added extra bounty for articles that involve attacks on custodians. This one is a scam, not an attack.

**Pull request merged**